fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveCacheUpdateProcessor.java

```diff
@@ -99,7 +98,7 @@ public void refreshTable(String dbName, Table table, boolean onlyCachedPartition
         if (table instanceof HiveMetaStoreTable) {
             HiveMetaStoreTable hmsTbl = (HiveMetaStoreTable) table;
             metastore.refreshTable(hmsTbl.getDbName(), hmsTbl.getTableName(), onlyCachedPartitions);
-            refreshRemoteFiles(hmsTbl.getTableLocation(), Operator.UPDATE, getExistPaths(hmsTbl), onlyCachedPartitions);
+            refreshRemoteFiles(table, Operator.UPDATE, getExistPaths(hmsTbl), onlyCachedPartitions);
```

Comment:
@Youngwb  @stephen-shelby  is it okay to remove table location, and use table here? And when we need table locaiton, we call `table.getTableLocation()`